content/editors/vim.md

```diff
@@ -1,4 +1,4 @@
-[Home](../../) > [Customize a Productive Coding Environment](../customize_your_environment.md) > Vim
+Back to [Customize a Productive Coding Environment](../customize_your_environment.md)
```

Comment:
I actually liked having the breadcrumb in there so users can navigate where they please. was trying to improve usability! Access to Home seems valuable to me still.